Std/Data/BitVec/Bitblast.lean

```diff
@@ -99,20 +96,21 @@ theorem adc_overflow_limit (x y i : Nat) (c : Bool) : x % 2^i + (y % 2^i + c.toN
   exact Nat.mod_lt _ (Nat.pow_two_pos i)
 
 theorem carry_succ (w x y : Nat) (c : Bool) : carry (succ w) x y c =
+    --
     decide ((x.testBit w).toNat + (y.testBit w).toNat + (carry w x y c).toNat ≥ 2) := by
```

Comment:
I'm sad that this isn't in terms of the at-least-two boolean operator `x && y || x && c || y && c`.

Should I be sad?

Maybe this is what I was saying above about `adc` being something new that I don't care about:

I would like the beginning of the file to define
`Nat.carry`, `Bool.atLeastTwo`, `Bool.xorThree`, and `BitVec.adc`.

Then I would like to have the theorems explaining how they relate via `BitVec.toNat` and `BitVec.getLsb`.

Finally (possibly in a different file!) I would like `+ = adc`.